src/main.ts

```diff
@@ -1,10 +1,12 @@
-
```

Comment:
The patch adds helpful comments to explain each step of the Vue app initialization process. This improves code readability and maintainability.

``` 
// Import necessary dependencies
import { createApp } from 'vue'
import App from './App.vue'
import router from './router'
```